### PR TITLE
docs: add "setting the language" example

### LINKED ISSUE #638: Add document for lang setting
It can be changed by setting `ctx.lang = "en"` in `main.ts`, but it is better to mention it in the documentation.

- Create FAQ section
- or, explicitly specify lang="en" in a generated main.ts

### PR DESCRIPTION
**Pull request by deer**
closes https://github.com/denoland/fresh/issues/638
Thanks to @hashrock for showing how to do this. Please let me know if the wording should be different.

### CODE REVIEW

**Review by hashrock — APPROVED** [review]
LGTM thanks!

File: docs/examples/setting-the-language.md
Comment (marvinhagemeister) [inline docs/examples/setting-the-language.md:6]: Nit: The directory argument is missing. Running this command as is will lead to an error. I'm hoping to make it optional with #1263 , but that's not merged yet.

Comment (deer) [reply docs/examples/setting-the-language.md:6]: Yes, I suppose I should have mentioned that when I opened this. I left it off, since I saw your 1263 and I figured the shortened version was clear enough. Perhaps this can be merged once the other one is merged.

Comment (marvinhagemeister) [reply docs/examples/setting-the-language.md:6]: Discard my comment. #1263 is merged 🎉

**Pull request merged**
